applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -550,181 +748,7 @@ static int initialize(le_audio_receive_cb recv_cb, le_audio_timestamp_cb timestm
-{
-	ARG_UNUSED(pa_sync);
-	ARG_UNUSED(broadcast_id);
-
-	LOG_WRN("%s not supported", __func__);
-	return -ENOTSUP;
-}
-
-void le_audio_conn_disconnected(struct bt_conn *conn)
-{
-	ARG_UNUSED(conn);
-
-	LOG_WRN("%s not supported", __func__);
-}
-
-int le_audio_ext_adv_set(struct bt_le_ext_adv *ext_adv)
-{
-	ARG_UNUSED(ext_adv);
-
-	LOG_WRN("%s not supported", __func__);
-	return -ENOTSUP;
-}
-
-void le_audio_adv_get(const struct bt_data **adv, size_t *adv_size, bool periodic)
-{
-	if (periodic) {
-		LOG_WRN("No periodic advertisement for CIS headset");
-		return;
-	}
-
-	*adv = ad_peer;
-	*adv_size = ARRAY_SIZE(ad_peer);
-}
-
-int le_audio_play(void)
-{
-	LOG_WRN("%s not supported", __func__);
-	return -ENOTSUP;
-}
-
-int le_audio_pause(void)
-{
-	LOG_WRN("%s not supported", __func__);
-	return -ENOTSUP;
-}
-
-int le_audio_send(struct encoded_audio enc_audio)
-{
-#if (CONFIG_BT_AUDIO_TX)
-	int ret;
-	struct net_buf *buf;
-	static bool hci_wrn_printed;
-
-	if (enc_audio.num_ch != 1) {
-		LOG_ERR("Num encoded channels must be 1");
-		return -EINVAL;
-	}
-
-	/* CIS headset only supports one source stream for now */
-	if (sources[0].stream->ep->status.state != BT_BAP_EP_STATE_STREAMING) {
-		LOG_DBG("Return channel not connected");
-		return 0;
-	}
-
-	/* net_buf_alloc allocates buffers for APP->NET transfer over HCI RPMsg,
-	 * but when these buffers are released it is not guaranteed that the
-	 * data has actually been sent. The data might be queued on the NET core,
-	 * and this can cause delays in the audio.
-	 * When stream_sent_cb() is called the data has been sent.
-	 * Data will be discarded if allocation becomes too high, to avoid audio delays.
-	 * If the NET and APP core operates in clock sync, discarding should not occur.
-	 */
-	if (atomic_get(&iso_tx_pool_alloc) >= HCI_ISO_BUF_ALLOC_PER_CHAN) {
-		if (!hci_wrn_printed) {
-			LOG_WRN("HCI ISO TX overrun");
-			hci_wrn_printed = true;
-		}
-
-		return -ENOMEM;
-	}
-
-	hci_wrn_printed = false;
-
-	buf = net_buf_alloc(iso_tx_pools[0], K_NO_WAIT);
-	if (buf == NULL) {
-		/* This should never occur because of the iso_tx_pool_alloc check above */
-		LOG_WRN("Out of TX buffers");
-		return -ENOMEM;
-	}
-
-	net_buf_reserve(buf, BT_ISO_CHAN_SEND_RESERVE);
-	net_buf_add_mem(buf, enc_audio.data, enc_audio.size);
-
-	atomic_inc(&iso_tx_pool_alloc);
-	ret = bt_bap_stream_send(sources[0].stream, buf, sources[0].seq_num++,
-				 BT_ISO_TIMESTAMP_NONE);
-	if (ret < 0) {
-		LOG_WRN("Failed to send audio data: %d", ret);
-		net_buf_unref(buf);
-		atomic_dec(&iso_tx_pool_alloc);
-		return ret;
-	}
-
-	return 0;
-#else
-	return -ENOTSUP;
-#endif /* (CONFIG_BT_AUDIO_TX) */
-}
-
-int le_audio_enable(le_audio_receive_cb recv_cb, le_audio_timestamp_cb timestmp_cb)
-{
-	int ret;
-
-	ret = initialize(recv_cb, timestmp_cb);
-	if (ret) {
-		LOG_ERR("Initialize failed");
-		return ret;
-	}
-
-	return 0;
-}
-
-int le_audio_disable(void)
+int unicast_server_disable(void)
```

Comment:
Probably we add a TODO in here for showing that we will take care the disable in later?

Reply:
Will remove for now.

Reply:
unicast_server_disable() can also be removed for now since it is not used